### ISSUE TITLE
Infinite loop during error report

### ISSUE THREAD

**Issue by Epskampie** [getsentry/sentry-symfony#119]
Hi, we recently faced an infinite loop caused by an error thrown while updating php files. This caused the same error to be logged thousands of times in sentry. Unfortunately it was hard to find the exact steps to reproduce for us, but we found a bug report on the symfony repo with more people with similar issues, including a possible explanation for the bug. Please see the thread here:

https://github.com/symfony/symfony/issues/26438

**Comment by Jean85** [comment]
This may also be related to https://github.com/getsentry/sentry-php/issues/552.
I'm currently investigating this, thanks for the pointer!

**Comment by Jean85** [comment]
https://github.com/symfony/symfony/pull/26568 fixes this issue. If you need to use it immediately, without waiting for the release, you can use my fork, that contains Symfony 3.4.6 + the cherry-picked fix.

To use the fork, you have to add to your `composer.json` this:
```json
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/jean85/symfony"
        }
    ],
```

and the change the requirement for the Symfony package like this:
```json
    "require": {
        "symfony/symfony": "dev-3-4-6-fatal-handler-fix as 3.4.6",
    }
```

Unfortunately, this fix enables a small hindrance: each fatal error is reported twice to Sentry, once as an `Error`, the second time as a Symfony's `FatalThrowableError`. I will try to solve this with a patch on my side, in the bundle.

**Comment by Jean85** [comment]
Yes! The [referenced PRs](https://github.com/symfony/symfony/pull/26568) has been included in the following Symfony releases:
 * 2.7.44
 * 2.8.37
 * 3.4.7
 * 4.0.7

Please update if you're experiencing this bug!
